website/docs/docs/collaborate/data-health-signals.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Data health signals"
+sidebar_label: "Data health signals"
+id: data-health-signals
+description: "Learn how data health signals offer a quick, at-a-glance view of data health when browsing your resources in dbt Explorer."
+image: /img/docs/collaborate/dbt-explorer/data-health-signal.jpg
+---
+
+# Data health signals <Lifecycle status="preview" />
+
+Data health signal offer a quick, at-a-glance view of data health when browsing your resources in dbt Explorer. These icons keep you informed on the status of your resource's health using the indicators **Healthy**, **Caution**, **Degraded**, or **Unknown**.
+
+- Supported resources are [models](/docs/build/models), [sources](/docs/build/sources), and [exposures](/docs/build/exposures).
+- For accurate health data, ensure the resource is up-to-date and had a recent job run.
+- Each data health signal reflects key data health components, such as test success status, missing resource descriptions, missing tests, absence of builds in 30-day windows, [and more](#data-health-signal-criteria)
+
+<Lightbox src="/img/docs/collaborate/dbt-explorer/data-health-signal.jpg" width="55%" title="View data health signals for your models."/> 
+
+## Access data health signals
+
+Access data health signals in the following places:
```

Comment:
For exposures, trust signals are also present in embedded health tiles

Reply:
that's a great question and i asked @rpourzand the same [in the comments earlier](https://github.com/dbt-labs/docs.getdbt.com/pull/6621#discussion_r1876287412)! this is what she said:

> Honestly, trust signal is more of a blanket statement for a number of things, including this.. I'd just call this - "Data Health Signals".

Reply:
@jcserv - do you see 'trust signal' and 'trust signal icons' mentioned in the docs? i thought i removed references to it?

Reply:
oh i see, you mean we say trust signal in the data health tile doc -- good flag!

Reply:
per Mirna's Slack note: 
> will add source collection and exposure update after meta 3973/3971 and 4025 are complete in a follow up pr